### PR TITLE
Update onboard-windows-multi-session-device.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the license requirement 

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11069

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

File: microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ Also, if you're using FSlogix user profiles, we recommend you exclude the follow
 
 #### Licensing requirements
 
-Note on licensing: When using Windows Enterprise multi-session, depending on your requirements, you can choose to either have all users licensed through Microsoft Defender for Endpoint (per user), Windows Enterprise E5, Microsoft 365 Security, or Microsoft 365 E5, or have the VM licensed through Microsoft Defender for Cloud.
+Note on licensing: When using Windows Enterprise multi-session, depending on your requirements, you can choose to either have all users licensed through Microsoft Defender for Endpoint (per user), Windows Enterprise E5,  Microsoft 365 E5 Security, or Microsoft 365 E5, or have the VM licensed through Microsoft Defender for Cloud.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/onboard-windows-multi-session-device.md]:
```suggestion
Note on licensing: When using Windows Enterprise multi-session, depending on your requirements, you can choose to either have all users licensed through Microsoft Defender for Endpoint (per user), Windows Enterprise E5, Microsoft 365 E5 Security, or Microsoft 365 E5, or have the VM licensed through Microsoft Defender for Cloud.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@tiburd it looks like not merged yet, can you please help. Thanks

**Comment by tiburd** [comment]
@Dansimp 
Hi Dan,
It looks like you intended to merge this PR, but I can't tell for sure.
Thanks!

**Review by v-britweston — APPROVED** [review]

**Pull request merged**